ts/components/NewRemindEmailValidationOverlay.tsx

```diff
@@ -46,34 +47,32 @@ import { IOToast } from "./Toast";
 import { LightModalContextInterface } from "./ui/LightModal";
 import { withLightModalContext } from "./helpers/withLightModalContext";
 import BaseScreenComponent from "./screens/BaseScreenComponent";
+import { CountdownProvider, useCountdown } from "./countdown/CountdownProvider";
 
-const emailSentTimeout = 10000 as Millisecond; // 10 seconds
+const emailSentTimeout = 10000 as Millisecond; // 60 seconds
```

Comment:
Watch out that I've left the debug values of `10000` instead of `60000` ;)

Reply:
done! 😁